odoo/addons/base/models/ir_ui_view.py

```diff
@@ -478,6 +478,7 @@ def _compute_warning_info(self):
 
     @api.model_create_multi
     def create(self, vals_list):
+        valid_types = [key for key, _ in self.fields_get()['type']['selection']]
```

Comment:
field_get compute a tons of stuff unused in this case. You can use Selection._selection instead: 
```suggestion
        valid_types = self._fields['type']._selection
```